content/60.methods.md

```diff
@@ -1,50 +1,101 @@
 We designed software to generate figures that automatically update using external data sources. <!-- TODO: replace figure reference with manuscript reference or reference figure conditionally based on template variable [, such as Figure @fig:csse-deaths,] -->
-Our primary goal is to sort and distill informative content out of the overwhelming flood of information [@doi:10.1038/s42254-020-0175-7] and help the broader scientific community become more conversant on this critical subject.
+Our primary goal was to provide the backend for scientists to sort and distill informative content out of the overwhelming flood of information [@doi:10.1038/s42254-020-0175-7] and help the broader scientific community become more conversant on this critical subject.
 Thus, our approach has been to develop a real-time, collaborative effort that welcomes submissions from scientists worldwide into this ongoing effort.
-This document represents the first snapshot, which aims to reflect the state of the field as of October, 2020.
-We plan to refine and expand this document until technologies to mitigate the pandemic are widely available.
+This case study demonstrates the value of open collaborative writing tools such as Manubot to emerging crises such as COVID-19.
+By allowing us to keep a record of the evolution of information over time and to create figures and in-line text that autoupdated alongside the state of the world, it revealed the particular value that Manubot holds for managing a rapid changes in scientific thought.
 
-In an effort to keep pace as new information about COVID-19 and SARS-CoV-2 becomes available, this project is an open, collaborative effort that invited contributions from the scientific community broadly, similar to previous efforts to develop collaborative reviews [@doi:10.1098/rsif.2017.0387; @url:https://greenelab.github.io/deep-review].
+### Methods
+
+#### Description of the Problem
+
+<!--To Do: Plot the number of pubs in CORD-19 over time?-->
+
+As many other efforts have described, the rate of publishing of formal manuscripts and preprints about COVID-19 was unprecedented [@doi:10.1053/j.ackd.2020.08.003].
+Additionally, the broader topic of COVID-19 intersects with a wide range of fields, including virology, immunology, medicine, pharmacology, evolutionary biology, public health, and more.
+Therefore, any effort to comprehensively document and evaluate this body of literature would require insight from scientists across a number of fields.
+Furthermore, during the initial phase of the COVID-19 pandemic during spring and summer 2020, and much longer in some part of the world, many biological scientists were unable to access their research spaces.
+As a result, early career researchers (ECR) and students were likely to lose out on valuable time for conducting experiments. <!--To Do: look at equity analyses of the effects on the pandemic to see if there is any data on this yet?-->
```

Comment:
Manubot provided a way for all contributors, including ECRs, to join a massive collaborative projects but also demonstrate their individual contributions to the larger work.